pkg/dev/dev.go

```diff
@@ -25,6 +29,7 @@ type DevClient struct {
 	portForwardClient portForward.Client
 	watchClient       watch.Client
 	bindingClient     binding.Client
+	filesystem        filesystem.Filesystem
 }
 
 var _ Client = (*DevClient)(nil)
```

Comment:
Could we ensure here that the `Handler` interface is implemented since it's no longer on the CLI but on the business layer?

Reply:
This Handler is not necessary anymore, removing it.